tachyon.php

```diff
@@ -55,7 +55,24 @@ function tachyon_url( $image_url, $args = [], $scheme = null ) {
 	}
 
 	$image_url = apply_filters( 'tachyon_pre_image_url', $image_url, $args, $scheme );
-	$args      = apply_filters( 'tachyon_pre_args', $args, $image_url, $scheme );
+
+	// If the image URL has X-Amz params for signed requests, we need to add them to the Tachyon URL
+	// under a `presign` param. However, only do this if we're on a version of TFA that supports it.
+	if ( getenv( 'TFA_VERSION' ) && version_compare( getenv( 'TFA_VERSION' ), '4.6.0', '>=' ) && stripos( $image_url, 'X-Amz-' ) !== false ) {
```

Comment:
Hmm. This looks very internal to Altis. Perhaps it's worth adding a boolean filter instead so other users/platforms can presign requests as well? And we can trigger on `TFA_VERSION` in altis/cloud. What do you think?

Reply:
hmm yeah, we could actually move all of the code to Altis Cloud in that case, and just filter the `tachyon_url`.

Reply:
Actually I think yeah it makes sense to have the logic here, so maybe it's better to define a `TACHYON_SERVER_VERSION` or a filter `tachyon_server_version` and keep the rest of the code here